yeesheen.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: page
+title: Yeesheen
+---
+
+## What I do
+I manage the Knowledge Base and in-app education programs at Optimizely (an A/B testing and experimentation platform)!
+
```

Comment:
Haha, great question. I actually wish there were another (more user-friendly) word we could use entirely! But I haven't seen "knowledgebase" much -- curious what you think. @kelly-o